components/browse-decks/WelcomeRepeat.js

```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components'
 
 const WelcomeContainer = styled.div`
-  display: inline;
```

Comment:
I think this placed the text _under_ the 'create deck' button. Was this on purpose?

Reply:
this didn't do anything